libc/test/src/math/explogxf_test.cpp

```diff
@@ -34,7 +36,7 @@ TEST_F(LlvmLibcExplogfTest, ExpInFloatRange) {
     return static_cast<float>(result.mh * r);
   };
   auto f_check = [](float x) -> bool {
-    return !((isnan(x) || isinf(x) || x < -70 || x > 70 ||
```

Comment:
this will also need to be fully qualified.  Maybe you can have `using FPBits = LIBC_NAMESPACE::fputil::FPBits<float>` at the beginning of the file, and the all subsequence usage can just be `FPBits`